fetch-from-ncbi-virus

```diff
@@ -21,27 +15,13 @@ bin="$(dirname "$0")"
 
 
 main() {
-    declare -a filters
-    declare -a fields
-
-    for arg; do
-        case "$arg" in
-            --filter=*)
-                filters+=("${arg#*=}")
-                shift;;
-            --field=*)
-                fields+=("${arg#*=}")
-                shift;;
-            *)
-                break;;
-        esac
-    done
-
     local ncbi_taxon_id="${1:?NCBI taxon id is required.}"
     local github_repo="${2:?A GitHub repository with owner and repository name is required as the second argument}"
+    declare -a ncbi_virus_url_options
+    ncbi_virus_url_options=("${@:3}")
 
     local ncbi_virus_url
-    ncbi_virus_url="$("$bin"/ncbi-virus-url --ncbi-taxon-id "$ncbi_taxon_id" --filters "${filters[@]}" --fields "${fields[@]}")"
+    ncbi_virus_url="$("$bin"/ncbi-virus-url --ncbi-taxon-id "$ncbi_taxon_id" "${ncbi_virus_url_options[@]}")"
```

Comment:
Hmm, I still get an error when I try to run with the ancient macOS Bash (3.2.57):
```
$ /bin/bash ./fetch-from-ncbi-virus 10244 nextstrain/monkeypox
./fetch-from-ncbi-virus: line 24: ncbi_virus_url_options[@]: unbound variable
```

But fine to leave as-is if we are going with you must upgrade Bash.

Reply:
Oops, I didn't test after adding the `ncbi_virus_url_options` variable. I [removed it](https://github.com/nextstrain/ingest/compare/34089c4c0659dfb55861c3e4a295cd8d5fc9c684..966ebcf0e9afc88674999bb5bb075c9ea6176ff3) so the change actually improves Bash compatibility.

Also added a test which initially fails on ancient Bash then succeeds after 966ebcf0e9afc88674999bb5bb075c9ea6176ff3.